sandbox/lua/decoders/linux_netdev.lua

```diff
@@ -0,0 +1,171 @@
+    | name:"eth0_receive_fifo" type:integer value:0
+    | name:"lo_transmit_bytes" type:integer value:50278
+    | name:"lo_receive_multicast" type:integer value:0
+    | name:"eth0_receive_packets" type:integer value:0
+    | name:"lo_transmit_compressed" type:integer value:0
+    | name:"eth0_transmit_packets" type:integer value:0
+    | name:"lo_transmit_colls" type:integer value:0
+    | name:"eth0_transmit_compressed" type:integer value:0
+    | name:"eth0_receive_drop" type:integer value:0
+    | name:"eth0_receive_frame" type:integer value:0
+    | name:"eth0_transmit_errs" type:integer value:0
+    | name:"eth0_transmit_fifo" type:integer value:0
+    | name:"lo_receive_drop" type:integer value:0
+    | name:"eth0_receive_bytes" type:integer value:0
+    | name:"lo_transmit_drop" type:integer value:0
+    | name:"lo_receive_frame" type:integer value:0
+    | name:"FilePath" type:string value:"/proc/net/dev"
+    | name:"lo_transmit_fifo" type:integer value:0
+    | name:"lo_transmit_errs" type:integer value:0
+    | name:"eth0_transmit_drop" type:integer value:0
+    | name:"lo_transmit_packets" type:integer value:601
+    | name:"lo_receive_compressed" type:integer value:0
+    | name:"lo_receive_fifo" type:integer value:0
+    | name:"lo_receive_errs" type:integer value:0
+    | name:"eth0_transmit_carrier" type:integer value:0
+    | name:"lo_receive_packets" type:integer value:601
+    | name:"lo_receive_bytes" type:integer value:50278
+    | name:"eth0_transmit_colls" type:integer value:0
+    | name:"eth0_receive_compressed" type:integer value:0
+    | name:"eth0_receive_errs" type:integer value:0
+    | name:"eth0_receive_multicast" type:integer value:0
+    | name:"eth0_transmit_bytes" type:integer value:0
+
+--]]
+
+local table = require "table"
+local l = require "lpeg"
+l.locale(l)
+
+local payload_keep = read_config("payload_keep")
+
+local space = l.P' '^0
+local keyword = (l.lower + l.digit)^1
+local counter = l.Ct(l.Cg(l.digit^1 / tonumber, "value") * l.Cg(l.Cc'2', "value_type"))
+local header = l.Ct(
+               l.P'Inter-'
+             * l.P'|'
+             * space
+             * l.P'Receive'
+             * space
+             * l.P'|'
+             * space
+             * l.P'Transmit'
+             * l.P'\n'
+             -- second line
+             * space
+             * l.P'face'
+             * space
+             * l.P'|'
+             * space
+             * l.Ct((l.Cg(keyword) * space)^1)
+             * l.P'|'
+             * space
+             * l.Ct((l.Cg(keyword) * space)^1)
+             * l.P'\n'
+             )
+local line = l.Ct(
+             space
+           * l.Cg(keyword)
+           * l.P':'
+           * space
+           * l.Ct((l.Cg(counter) * space)^1)
+           * l.P'\n'
+           )
+
+local grammar = l.Ct(
+                header
+              * l.Ct(line^1)
+              )
+
+local field_names = nil
+
+local msg = {
+    Timestamp = nil,
+    Type = "stats.netstat",
```

Comment:
Should this be `stats.netdev` to distinguish it from the messages generated by the other decoder?